analysis/cprs/generate_posterior.py

```diff
@@ -303,11 +303,14 @@
     ##Second wave inputs
     sec_states=sorted(['NSW','VIC'])
     sec_start_date = '2020-06-01'
-    if data_date > pd.to_datetime("2020-06-12"):
-        possible_end_date = data_date - timedelta(10)#subtract 10 days to aovid right truncation
-    else:
-        possible_end_date = pd.to_datetime("2020-06-01")
-    sec_end_date = possible_end_date.strftime('%Y-%m-%d')
+    sec_end_date = '2021-01-18'
```

Comment:
Just be mindful that this is a global end date on the second wave, and at line 348 there are state specific date ranges. Don't know what is a better solution, but might lead to conflicts later.